<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GDALConnectorDialogForm</class>
 <widget class="QDialog" name="GDALConnectorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GDAL Connector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">QWidget { background: white }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="m_datasourceTypeTitleLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>GDAL - Geospatial Data Abstraction Library</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Dataset access information</string>
       </property>
       <property name="buddy">
        <cstring>m_datasetLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <item>
        <widget class="QLineEdit" name="m_datasetLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="m_searchDatasetToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="m_fileRadioButton">
          <property name="text">
           <string>File</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_dirRadioButton">
          <property name="text">
           <string>Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_urlRadioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data Source Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="m_datasourceTitleLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data Source Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_datasourceDescriptionTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>A more detailed description about the data source</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
     <item>
      <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
       <property name="text">
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="m_testPushButton">
         <property name="text">
          <string>&amp;Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_openPushButton">
         <property name="text">
          <string>&amp;Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_closePushButton">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_fileRadioButton</tabstop>
  <tabstop>m_searchDatasetToolButton</tabstop>
  <tabstop>m_datasetLineEdit</tabstop>
  <tabstop>m_datasourceTitleLineEdit</tabstop>
  <tabstop>m_datasourceDescriptionTextEdit</tabstop>
  <tabstop>m_openPushButton</tabstop>
  <tabstop>m_closePushButton</tabstop>
  <tabstop>m_testPushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_closePushButton</sender>
   <signal>pressed()</signal>
   <receiver>GDALConnectorDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
